sale_secondary_salesperson/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Sale Secondary Salesperson',
+    'category': 'Customer Relationship Management',
+    'summary': """Sale Secondary Salesperson""",
```

Comment:
A summary with the same content of the name is useless. Remove it (and don't use triple quote for one liner string - except docstrings -).